proposals/lip-0057.md

```diff
@@ -113,11 +113,11 @@ For the rest of this proposal we define the following constants:
 | `MAX_NUMBER_STAKING_SLOTS` | uint32 | `10` | The maximum size of the stakes array of a staker substore entry. |
 | `MAX_NUMBER_PENDING_UNLOCKS` | uint32 | `20` | The maximum size of the pendingUnlocks array of a staker substore entry. |
 | `FAIL_SAFE_MISSED_BLOCKS` | uint32 | `50` | The number of consecutive missed blocks used in the fail safe banning mechanism. |
-| `FAIL_SAFE_INACTIVE_WINDOW` | uint32 | `130,000` | The length of the inactivity window used in the fail safe banning mechanism. |
-| `LOCKING_PERIOD_STAKING` | uint32 | `26,000` | The [locking period][lip-0023#explicit-unlock-mechanism] for regular staked tokens. |
-| `LOCKING_PERIOD_SELF_STAKING` | uint32 | `260,000` | The [locking period][lip-0023#explicit-unlock-mechanism] for self-staked tokens. |
-| `PUNISHMENT_WINDOW_STAKING` | uint32 | `260,000` | The punishment period for regular stakes, i.e., how many blocks after misbehavior report stakers can unlock tokens unstaked from a punished validator. |
-| `PUNISHMENT_WINDOW_SELF_STAKING` | uint32 | `780,000` | The punishment period for self-stakes, i.e., how many blocks after misbehavior report punished validators can unlock their self-unstaked tokens. |
+| `FAIL_SAFE_INACTIVE_WINDOW` | uint32 | 120960 = 14x24x360 | The length of the inactivity window used in the fail safe banning mechanism, measured in number of blocks. |
```

Comment:
```suggestion
| `FAIL_SAFE_INACTIVE_WINDOW` | uint32 | 120960 = 14 * 24 * 3600 // BLOCK_TIME | The length of the inactivity window used in the fail safe banning mechanism, measured in number of blocks. |
```

Reply:
I think using the block time for the Lisk Mainchain in addition makes the computation more transparent and would make it easier for sidechains with a different block time to compute the analogous constant value. Similar, below.

Reply:
Committed suggestion. Although I remember there was some issue about block time in SDK and somehow we were avoiding to use it, but I might be missing something.